Buildings/Fluid/HeatExchangers/Validation/WetCoilEffectivenessNTU.mo

```diff
@@ -90,58 +90,17 @@ model WetCoilEffectivenessNTU
     dp1_nominal=0,
     configuration=hexCon,
     show_T=true) "Effectiveness-NTU coil model (parameterized with nominal UA)"
-    annotation (Placement(transformation(extent={{-30,4},{-10,24}})));
+    annotation (Placement(transformation(extent={{-30,44},{-10,64}})));
   Sources.MassFlowSource_T souWat1(
     redeclare package Medium = Medium_W,
     m_flow=m1_flow_nominal,
     T=T_a1_nominal,
     nPorts=1)
     "Source for water"
-    annotation (Placement(transformation(extent={{-180,10},{-160,30}})));
-  WetCoilCounterFlow hexDis(
```

Comment:
See my comment at https://github.com/ibpsa/modelica-ibpsa/pull/1549#issue-1049864543.
The discretized model has been removed in IBPSA because it is not merged in the library. 
That gives a validation model different between Buildings and IBPSA (same holds true for the mos script).
I would like to avoid removing the discretized model from the validation model in Buildings because it provides a good reference.
How can we handle that?

Reply:
Done, but I renamed it `WetCoilEffectivenessNTUCounterFlow`. So we have 
```
Buildings/Fluid/HeatExchangers/Validation/
...
├── WetCoilEffectivenessNTUCounterFlow.mo
├── WetCoilEffectivenessNTUHeating.mo
└── WetCoilEffectivenessNTU.mo
```
and only `IBPSA/Fluid/HeatExchangers/Validation/WetCoilEffectivenessNTU.mo`.

Reply:
Thanks, that looks good.